tests/conftest.py

```diff
@@ -287,11 +287,21 @@ def init_persistence(default_conf):
     init_db(default_conf['db_url'], default_conf['dry_run'])
 
 
+@pytest.fixture(scope='function')
+def init_persistence_usdt(default_conf_usdt):
+    init_db(default_conf_usdt['db_url'], default_conf_usdt['dry_run'])
```

Comment:
this is pretty pointless ... it's a fixture that doesn't use anything you're changing (it's only using db_url and dry_run) ... so you can safely use the original fixture.